specification/appcomplianceautomation/resource-manager/readme.java.md

```diff
@@ -4,7 +4,8 @@ These settings apply only when `--java` is specified on the command line.
 
 ``` yaml $(java)
 client-flattened-annotation-target: disabled
+title: AppComplianceAutomationClient
 service-name: "App Compliance Automation"
 uuid-as-string: true
-stream-style-serialization: true
```

Comment:
Do we have issue enabling it?

Reply:
This is probably for you to decide.

Currently we have handwritten code using Jackson e.g.
```
    public String publicSettingsAsJsonString() {
        ObjectMapper mapper = new ObjectMapper();
        try {
            return mapper.writeValueAsString(this.publicSettings());
        } catch (JsonProcessingException jex) {
            return null;
        }
    }
```